package.json

```diff
@@ -20,60 +24,114 @@
     "mocha": "mocha --require intelli-espower-loader test/test.js --full-trace --check-leaks --exit",
     "test": "nyc npm run mocha"
   },
+  "scripts": {
+    "commitlint": "commitlint",
+    "commitmsg": "commitlint -e $GIT_PARAMS",
+    "lint": "eslint --cache lib test",
+    "ci:lint:commits": "commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}",
+    "lint-staged": "lint-staged",
+    "release": "standard-version",
+    "release:ci": "conventional-github-releaser -p angular",
+    "release:validate": "commitlint --from=$(git describe --tags --abbrev=0) --to=$(git rev-parse HEAD)",
+    "security": "nsp check",
+    "test": "NODE_TLS_REJECT_UNAUTHORIZED=0 jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "npm run test:cover && npm run test:cover:report",
+    "test:cover": "nyc jest --collectCoverageFrom='lib/**/*.js' --coverageReporters=none --coverage",
+    "test:cover:report": "nyc report --reporter=html",
+    "ci:lint": "npm run lint && npm run security",
+    "ci:test": "NODE_TLS_REJECT_UNAUTHORIZED=0 npm run test -- --runInBand",
+    "ci:coverage": "NODE_TLS_REJECT_UNAUTHORIZED=0 npm run test:coverage -- --runInBand",
+    "defaults": "webpack-defaults"
+  },
   "files": [
     "lib/",
-    "cli.js",
-    "index.js"
+    "schemas/",
+    "LICENSE",
+    "README.md"
   ],
   "peerDependencies": {
     "webpack": "^4.0.0"
   },
   "dependencies": {
     "@shellscape/koa-static": "^4.0.4",
+    "@webpack-contrib/cli-utils": "^1.0.1",
     "@webpack-contrib/config-loader": "^1.1.1",
+    "@webpack-contrib/schema-utils": "^1.0.0-beta.0",
     "chalk": "^2.3.0",
     "clipboardy": "^1.2.2",
     "cosmiconfig": "^5.0.2",
     "debug": "^3.1.0",
-    "find-up": "^2.1.0",
+    "decamelize": "^2.0.0",
     "get-port": "^3.2.0",
     "import-local": "^1.0.0",
+    "is-plain-obj": "^1.1.0",
     "killable": "^1.0.0",
     "koa": "^2.4.1",
-    "koa-webpack": "^4.0.0",
-    "lodash": "^4.17.5",
+    "koa-webpack": "^5.0.1",
     "loud-rejection": "^1.6.0",
+    "mem": "^3.0.0",
     "meow": "^5.0.0",
+    "merge-options": "^1.0.1",
     "nanobus": "^4.3.1",
+    "node-version": "^1.1.3",
     "opn": "^5.1.0",
+    "p-defer": "^1.0.0",
     "resolve": "^1.6.0",
+    "strip-ansi": "^4.0.0",
     "time-fix-plugin": "^2.0.0",
     "update-notifier": "^2.3.0",
-    "url-join": "3.0.0",
+    "url-join": "^4.0.0",
     "v8-compile-cache": "^2.0.0",
-    "webpack-hot-client": "^3.0.0",
```

Comment:
Now we should put `webpack-hot-client` in `package.json`?

Reply:
I don't believe thats needed because it's a dependency of `koa-webpack`